x-pack/filebeat/Jenkinsfile.yml

```diff
@@ -101,6 +101,20 @@ stages:
         platforms:             ## override default labels in this specific stage.
             - "windows-7"
         stage: extended
+    cloud:
+      cloud: "mage build test"
+      withModule: true       ## run the ITs only if the changeset affects a specific module.
+      dirs:                  ## run the cloud tests for the given modules.
+        - "x-pack/filebeat/input/awss3/_meta/terraform" # needed by startCloudTestEnv in order to apply terraform
+        - "x-pack/filebeat/input/awss3" # the actual dir where tests reside
```

Comment:
This looks a bit hacky, I would prefer if we don't need to add both directories.

I see some options:
* Add a `terraformDir` config to this `cloud` step, so we can explicitly configure where is the terraform scenario and where the tests. This would have the advantage of allowing to use the same scenario for tests residing in multiple scenarios.
* Add a terraform file where the tests reside, so we don't have to add both directories. This terraform file could use `_meta/terraform` as a module.
* odify `startCloudTestEnv` to use `./_meta/terraform` if there are no terraform files in the